code/modules/surgery/organs/lungs.dm

```diff
@@ -39,6 +39,10 @@
 	var/healium_para_min = 3
 	///Minimum amount of healium to knock you down for good
 	var/healium_sleep_min = 6
+	///Minimum amount of hexane needed to start having effect
+	var/hexane_min = 2
+	var/n2o_euphoria
+	var/healium_euphoria
```

Comment:
Why store them on the lung? Information from the last go around doesn't really need to persist, does it?

Reply:
moved to the proc now